Shared/Samples/Apply colormap renderer to raster/README.md

```diff
@@ -0,0 +1,38 @@
+# Apply colormap renderer to raster
+
+Apply a colormap renderer to a raster.
+
+![Image of apply colormap renderer to raster sample](apply-colormap-renderer-to-raster.png)
```

Comment:
```suggestion
![Image of Apply colormap renderer to raster sample](apply-colormap-renderer-to-raster.png)
```

Reply:
Does the capitalization make any practical difference? This is just alt text for an image, right?

Reply:
It doesn't matter much. Some sample capitalize while some doesn't